src/components/button/button.scss

```diff
@@ -31,6 +31,28 @@
   @include md-fab($md-mini-fab-size, $md-mini-fab-padding)
 }
 
+[md-icon-button] {
+  @extend %md-button-base;
```

Comment:
Yes, it is already covered by the default button base.

![image](https://cloud.githubusercontent.com/assets/4987015/13903697/cdc5c1b2-ee86-11e5-93ad-b4cfb474c92e.png)